requirements.txt

```diff
@@ -27,8 +26,6 @@ pexpect==4.1.0
 pathlib2==2.1.0
 pickleshare==0.7.2
 simplegeneric==0.8.1
-path.py==8.2.1
-backports.shutil_get_terminal_size==1.0.0
```

Comment:
We can put them back in. I eventually figured out that I could work around pip's issue with dots and underscores in the package name on a PyPI mirror by adding a matching directory with a '-' instead of a dot or underscore. So 'path-py' and 'backports-shutil-get-terminal-size' exist in our mirror now and pip can find them.